performance_manager/lib/rt_vehicle_positions.py

```diff
@@ -0,0 +1,277 @@
+import logging
+
+import pandas
+import numpy
+import sqlalchemy
+import re
+import datetime
+
+from sqlalchemy import select
+from sqlalchemy import update
+from sqlalchemy import insert
+from sqlalchemy import delete
+from sqlalchemy.orm import sessionmaker
+
+from typing import Optional
+from typing import Union
+from typing import List
+
+from .s3_utils import read_parquet
+from .postgres_utils import VehiclePositionEvents
+
+
+def get_vp_dataframe(to_load: Union[str, List[str]]) -> pandas.DataFrame:
+    """
+    return a dataframe from a vehicle position parquet file (or list of files)
+    with expected columns
+    """
+    vehicle_position_cols = [
+        "current_status",
+        "current_stop_sequence",
+        "stop_id",
+        "vehicle_timestamp",
+        "direction_id",
+        "route_id",
+        "start_date",
+        "start_time",
+        "vehicle_id",
+    ]
+
+    return read_parquet(to_load).loc[:, vehicle_position_cols]
```

Comment:
Load parquet file, or list of parquet files from `read_parquet` function and limit returned dataframe to relevant columns.

Reply:
can't we do this with the `filters` arg on `read_parquet`?

Reply:
I thought we would be able to, but based on my testing/reading it seems like filter so meant to do a pre-SELECT on data being pulled from the parquet files. Which I don't think would really help us.

`read_pandas` actually has a `columns` parameter, but I was messing around with that at the end of last week, and that is what was causing my extremely slow download times. 

The implementation I have here seems to download the dataset immediately. The only real downside is if the dataframe size in memory becomes an issue.